filament/backend/src/metal/MetalHandles.mm

```diff
@@ -841,34 +841,55 @@ BufferDescriptor b(nullptr, 0u, [](void* buffer, size_t size, void* user) {
         }
     }
 
-    Attachment depthAttachment = getDepthAttachment();
-    descriptor.depthAttachment.texture = depthAttachment.getTexture();
-    descriptor.depthAttachment.level = depthAttachment.level;
-    descriptor.depthAttachment.slice = depthAttachment.layer;
+    Attachment depthStencilAttachment = getDepthStencilAttachment();
+    descriptor.depthAttachment.texture = depthStencilAttachment.getTexture();
+    descriptor.depthAttachment.level = depthStencilAttachment.level;
+    descriptor.depthAttachment.slice = depthStencilAttachment.layer;
     descriptor.depthAttachment.loadAction = getLoadAction(params, TargetBufferFlags::DEPTH);
     descriptor.depthAttachment.storeAction = getStoreAction(params, TargetBufferFlags::DEPTH);
     descriptor.depthAttachment.clearDepth = params.clearDepth;
 
-    const bool automaticResolve = samples > 1 && depthAttachment.getSampleCount() == 1;
+    const auto hasStencil = (MTLPixelFormatDepth32Float_Stencil8 == depthStencilAttachment.getPixelFormat()) || (MTLPixelFormatDepth24Unorm_Stencil8 == depthStencilAttachment.getPixelFormat());
```

Comment:
nit: ~200 chars long line, would be nice to add a break
even more nit: `auto` for a `bool` just hides the type without reducing verbosity
```suggestion
    const bool hasStencil = (MTLPixelFormatDepth32Float_Stencil8 == depthStencilAttachment.getPixelFormat()) || 
                            (MTLPixelFormatDepth24Unorm_Stencil8 == depthStencilAttachment.getPixelFormat());
```

Reply:
fixed in [96d6ff1](https://github.com/shapr3d/filament/pull/58/commits/96d6ff120e6cb7e6eb5c5f37789eb0600043d643)